src/aat/resources/features/F-097/F-097.feature

```diff
@@ -0,0 +1,23 @@
+@F-097
+Feature: CCD User Profile Api :: GET /users
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-424
+  Scenario: should fetch user profile
+    When a successful call [to create user profiles] as in [Standard_User_Profiles_Creation_Data]
+    And a request is prepared with appropriate values
+    And it is submitted to call the [Retrieve user profiles] operation of [CCD User Profile]
```

Comment:
We should mention in a separate line that the request contains a jurisdiction id to more clearly describe the business logic of the scenario,

Reply:
I does not have to pass jurisdiction id hence the reason I did not include it. See here:
https://github.com/hmcts/ccd-user-profile-api/blob/master/src/main/java/uk/gov/hmcts/ccd/endpoint/userprofile/UserProfileController.java#L64-L66